formats/financial_releases_campaign/frontend/examples/financial_releases_campaign.json

```diff
@@ -0,0 +1,20 @@
+{
+  "content_id": "333e6c9d-aaf9-4f7d-98ce-3ce57a79802e",
+  "base_path": "/lloydsshares",
+  "title": "Financial releases campaign page",
+  "description": "Campaign page for financial press releases",
+  "locale": "en",
+  "public_updated_at": "2015-12-07T11:22:21.000Z",
+  "format": "financial_releases_campaign",
+  "details": {
+    "image": {
+      "alt_text": "HM Government",
+      "url": "campaign.large.jpg"
+    },
+    "header_legal_disclaimer": "Header legal disclaimer Vel ut iudico dolore. Vix ne diam iuvaret referrentur, eam nisl accusamus temporibus ne. Nec quot deserunt dissentiet ut, ad  be available from <a href='/lloydsshares'>www.gov.uk/lloydsshares</a>.",
+    "title": "An Campaign Title",
```

Comment:
Yes absolutely. The idea is that the root of the document is for properties that are common to more or less everything, `details` is for things which vary by format.

If you want an example of an app that renders from content-store, [government-frontend](https://github.com/alphagov/government-frontend) is (hopefully!) relatively easy to follow.